src/e2e-app/index.html

```diff
@@ -1,6 +1,6 @@
 <!doctype html>
 {{!-- This file is an handlebar file that gets filled at build time. --}}
-<html>
+<html lang="en">
```

Comment:
You could probably use this PR for the a11y test fixes instead of #654 (because `lang="en"` on the demo-app will fix one)